### DISCUSSION TITLE
Existe algum projeto de integração com a LangChain?

### DISCUSSION THREAD

**Discussion by nascin** [maritaca-ai/maritalk-api#38] [Ideas]
Tenho alguns projetos uso langchain e openai, gostaria de testar a maritaca.

Tem alguma programação para integração com a LangChain?

**Comment by rodrigo-f-nogueira** [comment]
Olá @nascin, temos um google colab mostrando como usar a MariTalk com o LangChain:

https://colab.research.google.com/drive/1bka6wdgYQ9slKuNDRigBep4UY4HLhmHL?usp=sharing

**Reply by nascin** [reply]
Incrível! Com isso já consigo usar.

Vocês pensam em abrir um PR pra implementação nativa na langchain ou lanchsin comunity?

**Reply by rodrigo-f-nogueira** [reply]
Oi @nascin , vamos fazer isso, sim!

**Comment by hugoabonizio** [comment]
Olá @nascin! Segue a documentação sobre a integração com LangChain: https://python.langchain.com/docs/integrations/chat/maritalk/
